deploy/helm/grafana-operator/templates/deployment.yaml

```
@@ -20,6 +20,9 @@ spec:
       labels:
         {{- include "grafana-operator.selectorLabels" . | nindent 8 }}
```

The only change needed is to use the `grafana-operator.labels` macro here, as it would include the additional labels.

```suggestion
        {{- include "grafana-operator.labels" . | nindent 8 }}
```

---

Thanks for taking the time to re-contextualize this. I'm fine with merging something like this. Will discuss this with the other maintainers

---

I'm also happy to create a PR with this proposed change, if it helps move this along!

---

I'm happy to close this PR and have you open a new PR with this change

---

If either of you have the time, feel free to create the PR. Would close this PR in favour of the follow-up

---

New PR here: https://github.com/grafana/grafana-operator/pull/1649